### PR TITLE
Enable the upload button after opening a config

### LINKED ISSUE #1000: Upload config button doesn't activate after opening a config
**Describe the bug**
Upload config button doesn't activate after opening a config

**Steps To Reproduce**
1. Go to `Mobiflight Modules`
2. Select one of your boards
3. Open a config file

Button

**Expected behavior**
The `Upload config` button activates so you can upload the config.

**Actual behavior**
The `Upload config` button doesn't activate

**Details**
 - MobiFlight Version: 9.6.0

**Additional context**
If you expand the module and select any of the devices that will cause the button to activate.

### PR DESCRIPTION
**Pull request by neilenns**
Fixes #1000 

Enable the upload button whenever a config is loaded successfully.

### CODE REVIEW

File: UI/Panels/Settings/MobiFlightPanel.cs [hunk 1]
```diff
@@ -848,6 +848,7 @@ private void openToolStripButton_Click(object sender, EventArgs e)
 
                 moduleNode.ImageKey = "Changed";
                 moduleNode.SelectedImageKey = "Changed";
+                uploadToolStripButton.Enabled = true;
```
Comment (DocMoebiuz) [inline UI/Panels/Settings/MobiFlightPanel.cs:851]: there is nothing in the code that disables the button on opening inside this function. That's why I find the proposed solution doesn't fix the problem.

In your screen recoding the device is not selected by the user, but when you click the open button- i would guess that "afterselect"-event is not properly triggered in this case.

So the real fix should be to properly "select" the node and trigger the `mfModulesTreeView_AfterSelect` correctly.

Expected behavior:
* If no board is selected, the toolbar should be deactivated.
* If one board is selected, the toolbar items become active.
* If I open a new config, the board should stay selected as active, and all buttons in the toolbar therefore active too.

**Review by DocMoebiuz — CHANGES_REQUESTED** [review]
see comment

**Comment by DocMoebiuz** [comment]
Possibly related to #925

**Comment by neilenns** [comment]
@DocMoebiuz I agree the fix in this PR isn't great. I couldn't find a good way to fire the actual existing method for when selection changes. I tried forcing the selection to the active board but that didn't work. Am open to other ideas, or if you want to try coding up a fix.

**Comment by DocMoebiuz** [comment]
I don't want to spend too much effort on this but I wonder why the UI doesn't behave correctly in the first place. At what circumstance is it greying out the button? I looked through the AfterSelect method - as soon as the board is a MF board and gets selected then the toolbar should become enabled.

We have the other issue #925 where this could be the same root cause that triggers this behavior.

To reproduce this, what is needed?

A fresh MF board with no devices?
what else?

**Comment by neilenns** [comment]
@DocMoebiuz My suspicion is one of these two situations are what makes it happen:

* Only one board connected
* The board connected has no devices on it yet

Haven't tested it deeply to find out which case causes it though.

When I was debugging I did confirm that AfterSelect is triggered when I first open the dialog, but it never gets called again after I click on the Open button. Forcing it to get called after the Open dialog closes is a good solution but I couldn't get that to work.

**Pull request closed without merging**